packages/strapi-types/index.ts

```diff
@@ -31,4 +39,14 @@ export type Strapi = {
     [key: string]: any;
   };
   query: (s: string) => any;
+  connections: {
+    default: Knex;
+  };
+  models: {
+    [key: string]: Model;
+  };
+  db: any;
+  app: {
+    use: (x: (ctx: Context, next: Next) => Promise<any>) => any;
```

Comment:
middleware doesn't return void in that case? 🤔

Reply:
thats correct!